tests/testthat/test-epidatacall.R

```diff
@@ -0,0 +1,60 @@
+test_that("fetch_tbl", {
+  epidata_call <- covidcast(
+    data_source = "jhu-csse",
+    signals = "confirmed_7dav_incidence_prop",
+    time_type = "day",
+    geo_type = "state",
+    time_values = epirange("2020-06-01", "2020-08-01"),
+    geo_values = "ca,fl"
+  )
+  # Generated with
+  # epidata_call %>%
+  # fetch_debug(format_type = "classic") %>%
+  # readr::write_rds(testthat::test_path("data/test-classic.rds"))
+  mockery::stub(fetch_classic, "httr::content", readRDS(testthat::test_path("data/test-classic.rds")))
+  mockery::stub(fetch_tbl, "fetch_classic", fetch_classic)
```

Comment:
note: just the single `stub` wasn't working, at least on my system.  See a little discussion at https://github.com/cmu-delphi/epidatr/pull/99/commits/4b6f964beb1aefb350b6bd7d540f9d33416d02b1 and https://github.com/cmu-delphi/epidatr/issues/110.